aws-lambda-layer/Rakefile

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+BUILD_DIR = File.join(__dir__, 'build')
+
+desc 'Build FreeTDS AWS Lambda Layer'
+task :build, [:freetds_version] do |_t, args|
+  version = args[:freetds_version] || ENV['FREETDS_VERSION'] || '1.1.6'
```

Comment:
can you pull `1.1.6` out to a top level constant?

Reply:
Sure thing.